lara-typescript/package.json

```diff
@@ -6,11 +6,12 @@
     "publish:plugin-api": "npm run build && cd dist/plugin-api && npm publish --access public",
     "publish:interactive-api-client": "npm run build && cd dist/interactive-api-client && npm publish --access public",
     "build": "npm-run-all lint:build clean build:webpack build:docs",
-    "build:webpack": "webpack --devtool false && cp ./dist/lara-typescript.js ../app/assets/javascripts/ && cp ./dist/lara-typescript.css ../app/assets/stylesheets/",
+    "build:webpack": "webpack --devtool false && npm run build:copy-to-rails",
     "build:docs": "npm-run-all build:docs:plugin-api build:docs:interactive-api-client",
     "build:docs:plugin-api": "typedoc --readme src/plugin-api/README.md --out ../docs/lara-plugin-api src/plugin-api/index.ts",
     "build:docs:interactive-api-client": "typedoc --readme src/interactive-api-client/README.md --out ../docs/interactive-api-client src/interactive-api-client/index.ts",
-    "build:dev:lara-typescript": "webpack && cp ./dist/lara-typescript.js ../app/assets/javascripts/ && cp ./dist/lara-typescript.css ../app/assets/stylesheets/",
+    "build:dev:lara-typescript": "webpack && npm run build:copy-to-rails",
+    "build:copy-to-rails": "cp ./dist/lara-typescript.js ../app/assets/javascripts/ && cp ./dist/lara-typescript.css ../app/assets/stylesheets/ && cp -r ./dist/example-interactive ../public/",
```

Comment:
I think you should add to this:

`&& rm ../public/example-interactive/*.d.ts"` to avoid copying the `*.d.ts` files and then delete the `d.ts` files from this PR.

Reply:
I think I address this now.